### ISSUE TITLE
three-finger drag function

### ISSUE THREAD

**Issue by woody1122001** [imbushuo/mac-precision-touchpad#291]
Hi, great driver, nice and smooth, just like when i'm using mac.

Just wondering if you could also add an option for the three finger drag? it was there in the old macbooks but now it should be in the accessibility section of the mac setting. I'm used to using 3 fingers to drag and highlight words.

Thanks a lot!

**Comment by imbushuo** [comment]
It's current not possible to implement such function due to Windows limitation. Please file a feedback in Feedback Hub.

**Comment by shikatasoa** [comment]
Hi, great driver, 

"three-finger drag” is realized by "TrackPad ++".
So can it be implemented with "precision-touchpad"?
will be hoping‼
